extensions/mssql/resources/light/default_server.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
As mentioned above let's not put the default server icons here, they should stay inside our app and be the fallback no matter what.

Reply:
fixed.